share/spice/nobel_prize/footer.handlebars

```diff
@@ -0,0 +1,7 @@
+ {{#if hasPrizeCategory}}
+    <span class="one-line text--secondary" >Year : <b>{{year}}</b></span><br/> 
```

Comment:
Actually, as I mentioned before, we should remove `Year : ` and `Category : ` entirely. The footer should just be the year OR the category. Only one needs to be specified because he other will always be known based on the query